### PR TITLE
Bluetooth: tester: Add Mesh IV Update related commands

### PR DESCRIPTION
**Pull request by MariuszSkamra**
This adds Mesh IV Update related command handlers implementation.

Signed-off-by: Mariusz Skamra <mariusz.skamra@codecoup.pl>

### CODE REVIEW

File: tests/bluetooth/tester/src/bttester.h [hunk 1]
```diff
@@ -708,6 +708,16 @@ struct mesh_input_string_cmd {
 	u8_t string[0];
 } __packed;
 
+#define MESH_IVU_TEST_MODE		0x08
+struct mesh_ivu_test_mode_cmd {
+	u8_t toggle;
```
Comment (sjanc) [inline tests/bluetooth/tester/src/bttester.h]: name this param enable

File: tests/bluetooth/tester/src/bttester.h [hunk 2]
```diff
@@ -708,6 +708,16 @@ struct mesh_input_string_cmd {
...
+} __packed;
+
+#define MESH_IVU_TOGGLE_STATE		0x09
+struct mesh_ivu_toggle_state_cmd {
+	u8_t toggle;
```
Comment (sjanc) [inline tests/bluetooth/tester/src/bttester.h]: I think this command shouldn't take any parameter,  toggle by definition is changing bit value and tester doesn't really has means to to control it.

Just update BTP spec if needed with those.

**Review by sjanc — CHANGES_REQUESTED** [review]

File: tests/bluetooth/tester/src/bttester.h (hunk 2 above)
Comment (MariuszSkamra) [reply tests/bluetooth/tester/src/bttester.h]: done

File: tests/bluetooth/tester/src/bttester.h (hunk 1 above)
Comment (MariuszSkamra) [reply tests/bluetooth/tester/src/bttester.h]: done

**Review by sjanc — APPROVED** [review]

**Review by jhedberg — APPROVED** [review]

**Pull request merged**